MoonCrystal/MoonCrystal/SelectCapacity/CustomNumPad.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomNumpad.swift
+//  MoonCrystal
+//
+//  Created by seonu kim on 8/5/24.
+//
+
+import SwiftUI
+
+struct CustomNumPad : View {
+    @Binding var string : String
+    let maxDigits : Int = 18
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            VStack(alignment: .leading, spacing: 6) {
+                NumPadRow(keys: ["1", "2", "3"], action: keyWasPressed(_:))
+                NumPadRow(keys: ["4", "5", "6"], action: keyWasPressed(_:))
+                NumPadRow(keys: ["7", "8", "9"], action: keyWasPressed(_:))
+                NumPadRow(keys: ["", "0", "⌫"], action: keyWasPressed(_:))
+            }
+            .padding(.top, 6)
+            Spacer()
+        }
+        .frame(height: 291.0)
+        .background(Color(red: 0.81, green: 0.82, blue: 0.84))
+    }
+    
+    private func keyWasPressed(_ key: String) {
+        if ( string.trimmingCharacters(in: .whitespacesAndNewlines).count > maxDigits-1 && key != "⌫" ) {
```

Comment:
이렇게 쓰는게 조금 더 깔끔해보여요!
```suggestion
        if string.trimmingCharacters(in: .whitespacesAndNewlines).count > (maxDigits-1 && key != "⌫") {
```